.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {

```

Comment:
Here's the [documentation](https://eslint.org/docs/user-guide/configuring) for configuring ESLint